what-is.asciidoc

```diff
@@ -26,7 +26,7 @@ The components of an open, public, blockchain are (usually):
 * A consensus algorithm (e.g. Proof-of-Work) that decentralizes control over the blockchain, by forcing participants to compete and constraining them by the consensus rules.
 * One or more open-source software implementations of the above.
 
-All or most of these components are usually combined in a single software client. For example, in bitcoin, the reference implementation is developed by the _Bitcoin Core_ open source project, implemented as the +bitcoind+ client. In Ethereum, rather than a reference implementation, there is a _reference specification_, a mathematical description of the system in the <<yellowpaper>>. There are a number of clients, which are built according to the reference specification.
+All or most of these components are usually combined in a single software client. For example, in Bitcoin, the reference implementation is developed by the _Bitcoin Core_ open source project, implemented as the +Bitcoind+ client. In Ethereum, rather than a reference implementation, there is a _reference specification_, a mathematical description of the system in the <<yellowpaper>>. There are a number of clients, which are built according to the reference specification.
```

Comment:
Bitcoind is small b, it's case sensitive command, not a proper noun

Reply:
Same here, as a general pattern, I'd recommend avoid book code refactorization.

It is vital to put yourself in the place of the reader who tries to learn with the code offered as an example in the book. This should always work and faithfully reproduce the operations proposed in the content of the book.